### ISSUE TITLE
AsMavenExtension should use buildDir variable

### ISSUE THREAD

**Issue by thomasmhofmann** [diffplug/goomph#9]
The p2asmaven directory is created in a directory being determined like this:

`Object destination = "build/p2asmaven";`

In my multip-project build I am setting the buildDir like this:

```
	/*
	 * The buildDir should be in the build project instead of in each separate project.
	 */
	buildDir = new File(rootProject.projectDir, "build/" + project.name)

```
The p2asmaven directory ends up in the subproject's directory. 

The class AsMavenExtension could also use the buildDir variable from the project to determine the target location.

**Comment by nedtwigg** [comment]
I agree this change should be made, but in the meantime you can override it as such:

```
p2asmaven {
    destination 'whereveryouwant'
}
```

Out of curiosity, what did you rename your build directory to and why?

**Comment by thomasmhofmann** [comment]
I just wanted to have all build directories of all subprojects in the root project. This way it easier for me to pick the build artefacts and only one directory is changed during the build.

**Comment by nedtwigg** [comment]
Gotcha.  That seems like a pretty good reason.

**Comment by nedtwigg** [comment]
Released in 3.2.0.
